src/test/java/teammates/test/cases/logic/EmailSenderTest.java

```diff
@@ -82,15 +85,18 @@ public void testConvertToSendgrid() {
     @Test
     public void testConvertToMailgun() {
         EmailWrapper wrapper = getTypicalEmailWrapper();
-        FormDataMultiPart formData = new MailgunService().parseToEmail(wrapper);
-
-        assertEquals(wrapper.getSenderName() + " <" + wrapper.getSenderEmail() + ">",
-                     formData.getField("from").getValue());
-        assertEquals(wrapper.getRecipient(), formData.getField("to").getValue());
-        assertEquals(wrapper.getBcc(), formData.getField("bcc").getValue());
-        assertEquals(wrapper.getReplyTo(), formData.getField("h:Reply-To").getValue());
-        assertEquals(wrapper.getSubject(), formData.getField("subject").getValue());
-        assertEquals(wrapper.getContent(), formData.getField("html").getValue());
+        try (FormDataMultiPart formData = new MailgunService().parseToEmail(wrapper)) {
+
+            assertEquals(wrapper.getSenderName() + " <" + wrapper.getSenderEmail() + ">",
+                    formData.getField("from").getValue());
+            assertEquals(wrapper.getRecipient(), formData.getField("to").getValue());
+            assertEquals(wrapper.getBcc(), formData.getField("bcc").getValue());
+            assertEquals(wrapper.getReplyTo(), formData.getField("h:Reply-To").getValue());
+            assertEquals(wrapper.getSubject(), formData.getField("subject").getValue());
+            assertEquals(wrapper.getContent(), formData.getField("html").getValue());
+        } catch (IOException e) {
+            fail("Could not close email wrapper stream: " + TeammatesException.toStringWithStackTrace(e));
```

Comment:
Instead of catching the Exception and failing the test anyway, just throw the Exception as part of the method.